node/core/backing/src/lib.rs

```diff
@@ -1236,6 +1239,54 @@ async fn seconding_sanity_check<Context>(
 	SecondingAllowed::Yes(membership)
 }
 
+/// Performs seconding sanity check for an advertisement.
+#[overseer::contextbounds(CandidateBacking, prefix = self::overseer)]
+async fn handle_seconding_check_message<Context>(
+	ctx: &mut Context,
+	state: &mut State,
+	request: SecondingCheckRequest,
+	tx: oneshot::Sender<bool>,
+) {
+	let relay_parent = request.candidate_relay_parent;
+	if state
+		.per_relay_parent
+		.get(&relay_parent)
+		.map_or(false, |pr_state| pr_state.prospective_parachains_mode.is_enabled())
+	{
+		let result = seconding_sanity_check(
+			ctx,
+			&state.per_leaf,
+			&state.implicit_view,
+			request.candidate_hash,
+			request.candidate_para_id,
+			request.parent_head_data_hash,
+			relay_parent,
+		)
+		.await;
+
+		// If seconding sanity check doesn't pass now, fetching collation
+		// is pointless as it will also fail once candidate validation concludes.
+		// However, the response may happen to be empty: this means that the
+		// parent node-candidate has not been seconded yet. This is OK,
+		// the request will be kept in memory.
+		match result {
+			SecondingAllowed::No => {
+				let _ = tx.send(false);
+			},
+			SecondingAllowed::Yes(membership) =>
+				if membership.iter().any(|(_, m)| !m.is_empty()) {
+					let _ = tx.send(true);
+				} else {
+					state
+						.seconding_check_requests
+						.entry((request.candidate_para_id, request.parent_head_data_hash))
+						.or_default()
+						.push(PerSecondingCheckRequest { relay_parent, tx });
+				},
+		}
+	}
```

Comment:
`tx` is silently dropped here if the relay parent is unknown or does not support async backing. That is fine, but contrary to the documentation on the message